apps/nowcasting-app/components/logo.tsx

```diff
@@ -0,0 +1,10 @@
+export const OCFlogo = () => (
+  <a
+    className="py-[4px] z-20"
+    href="https://www.openclimatefix.org/"
+    target="_blank"
+    rel="noreferrer"
+  >
+    <img src="/OCF_icon_wht.svg" alt="ofc" width={60} />
+  </a>
+);
```

Comment:
good idea to put this in its own file

Reply:
everything looks good to me except for the tooltip pop-ups that peter pointed out.